client/lib/ui/Main.js

```diff
@@ -304,6 +306,15 @@ export default React.createClass({
               </div>}
               {pmNotices.map(pm => <PMNotice key={pm.get('kind') + pm.get('id')} pmId={pm.get('id')} nick={pm.get('nick')} kind={pm.get('kind')} />) }
               {this.state.update.get('ready') && <FastButton className="update-button" onClick={update.perform}><p>update ready<em>{Heim.isTouch ? 'tap' : 'click'} to reload</em></p></FastButton>}
+              {this.state.donations.eligible && this.state.donations.url && this.state.ui.notices.has('donations') && <div className="notice dark donations">
+                <div className="content">
+                  <span className="title">euphoria maintenance and development is enabled by</span>
```

Comment:
I find the mix of sentence and button in the notification to be confusing to parse when I first see it. The button calls my attention first, and then I read the label. Maybe "euphoria maintenance and development depends on you [support our patreon]"?